.github/libs/GitUtils.ts

```diff
@@ -1,15 +1,12 @@
+import * as core from '@actions/core';
 import {execSync, spawn} from 'child_process';
 import CONST from './CONST';
+import GithubUtils from './GithubUtils';
+import type {CommitType} from './GithubUtils';
 import sanitizeStringForJSONParse from './sanitizeStringForJSONParse';
 import {getPreviousVersion, SEMANTIC_VERSION_LEVELS} from './versionUpdater';
 import type {SemverLevel} from './versionUpdater';
 
-type CommitType = {
-    commit: string;
-    subject: string;
-    authorName: string;
-};
```

Comment:
This could remain until the refactor that removes git log, but I got this out the way sooner rather than later.